backend/src/main/java/quarano/diary/DiaryEntryReminderMailJob.java

```diff
@@ -0,0 +1,93 @@
+package quarano.diary;
+
+import io.vavr.control.Try;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import quarano.account.DepartmentRepository;
+import quarano.core.EmailSender;
+import quarano.core.EmailTemplates.Keys;
+import quarano.core.EnumMessageSourceResolvable;
+import quarano.tracking.ContactTypeLookup;
+import quarano.tracking.TrackedPerson;
+import quarano.tracking.TrackedPerson.TrackedPersonIdentifier;
+import quarano.tracking.TrackedPersonEmail;
+import quarano.tracking.TrackedPersonRepository;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.support.MessageSourceAccessor;
+import org.springframework.data.util.Streamable;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * This class collects the persons with missing diary entries and sends a reminder mail to every person.
+ *
+ * @author Jens Kutzsche
+ * @author Oliver Drotbohm
+ */
+@Component
+@Slf4j
+@RequiredArgsConstructor
+@ConditionalOnProperty(prefix = "quarano.diary", name = "reminder.enable", matchIfMissing = true)
+class DiaryEntryReminderMailJob {
+
+	private final @NonNull DiaryManagement diaries;
+	private final @NonNull EmailSender emailSender;
+	private final @NonNull TrackedPersonRepository persons;
+	private final @NonNull DepartmentRepository departments;
+	private final @NonNull MessageSourceAccessor messages;
+	private final @NonNull ContactTypeLookup lookup;
+
+	@Transactional
+	@Scheduled(cron = "0 10 12,23 * * *")
+	void checkForReminderMail() {
+
+		if (emailSender.testConnection().isFailure()) {
+			return;
```

Comment:
Extended in develop branch https://github.com/quarano/quarano-application/compare/21e1dae473ed...39b33cb3610e, because this was already be merged. Thank you for your hint.